CefSharp/WebBrowserExtensions.cs

```diff
@@ -445,7 +445,7 @@ public static Task<LoadUrlAsyncResponse> LoadUrlAsync(IChromiumWebBrowserBase ch
                 chromiumWebBrowser.LoadingStateChanged -= loadingStateChangeHandler;
 
                 //Ensure our continuation is executed on the ThreadPool
-                //For the .Net Core implementation we could use
+                //For the .Net implementation we could use
                 //TaskCreationOptions.RunContinuationsAsynchronously
```

Comment:
Need to quickly double check, I think these comments are actually outdated and no longer required. Pretty sure most of them now use `TaskCreationOptions.RunContinuationsAsynchronously`